src/resources/parser/scene.js

```diff
@@ -50,7 +54,6 @@ Object.assign(pc, function () {
             entity.setLocalEulerAngles(r[0], r[1], r[2]);
             entity.setLocalScale(s[0], s[1], s[2]);
             entity._enabled = data.enabled !== undefined ? data.enabled : true;
-            entity._enabledInHierarchy = entity._enabled;
```

Comment:
As we have talked about already this breaks that unit test which is problematic so we need to figure what causes it to fail.